changelog/3247.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed all transformations to properly handle a change in observation time.
```

Comment:
can you add a second changelog entry for the velocities stuff.

Reply:
Done